src/scss/_nav.scss

```diff
@@ -1,12 +1,12 @@
 @if $o-header-is-silent == false {
 
-	// This is applied to the <body> via JS, this does not comply
-	// with the Origami spec, and issue #129 has been raised
+	// This is applied to the o-header element via JS
 	// It's used to avoid scrolling when the meganav
 	// is open in a mobile view
 	.o-header--mega-nav-open {
```

Comment:
Oh, true! You're applying this to the header element, sorry! Ignore me. If as Rhys says, this works on Next, fine with me to release it :shipit: